release-tasks.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/zsh
+
+python manage.py db stamp head
+python manage.py db migrate
+python manage.py db upgrade
```

Comment:
@brittanylau I think I made a change to the migrations folder that messed up the state of our Heroku deployment, but I'm hoping `python manage.py db stamp head` will fix it. (If not I'll keep investigating.)

Reply:
Sounds good! Thank you 😄

Reply:
Did you get an error that looked like "migration 2dndso3ta3 not found" or something? I had an error like that on my localhost, and I had to go into `psql freefrom_map_dev` and run `DROP TABLE alembic_version;` to fix it

Reply:
Yes I did! I ended up connecting to the Heroku dyno to fix the issue.